setup.cfg

```diff
@@ -20,5 +20,15 @@ ignore =
     # Ambiguous variable names
     E741,
     # Allow breaks after binary operators
-    W504
+    W504,
+    # Include name with TODOs as in # TODO(yourname)
+    H101,
+    # Do not import more than one module per line
+    H301,
+    # Alphabetically order imports by the full module path
+    H306,
+    # Multi line docstrings should start without a leading new line
+    H404,
+    # Multi line docstrings should start with a one line summary followed by an empty line
+    H405
```

Comment:
I added a few more of these in https://github.com/jupyter/enterprise_gateway/pull/908/commits/d1c1d91bb6c460c0ee8759fc41f89dd333829cb1.

That PR also changes from nose to pytest for the driver (since pytest run nosetests, but not the other way around).

Reply:
Yes, you are ignoring too much... I synced the ignores with what we have for Elyra... and fixed the rest.

Reply:
ok - good idea, I should have just pulled from E.